Makefile

```diff
@@ -100,22 +99,44 @@ lint-and-type-check: ## Lint your code
 	interrogate superduperdb
 
 
-##@ Demo Environment
+test-notebooks: ## Test notebooks (arg: NOTEBOOKS=<path_to_files>)
+	@if [ -n "${NOTEBOOKS}" ]; then	\
+		pytest --nbval ${NOTEBOOKS}; 	\
```

Comment:
Does this work?

I'm not sure that `pytest` can be used to run notebooks

Reply:
yeap. it's the `--nbval` that  does all the job.